src/utilities/modal.ts

```diff
@@ -441,6 +478,47 @@ export async function showPlanMergeRequestsModal(
   });
 }
 
+/**
+ * Shows a RestorePlanSnapshotModal with the supplied arguments.
+ */
+export async function showRestorePlanSnapshotModal(
```

Comment:
Is this for the next PR? Cool to keep it in but just wanted to make sure it wasn't currently accessible?

Reply:
Yep! I tried to hopefully set you up as much as possible for the panel work.

Reply:
:keanu-thanks: